src/global/project/mod.rs

```diff
@@ -662,8 +662,22 @@ impl Project {
         Ok(state_changes)
     }
 
-    /// Get all installed executables for specific environment.
-    pub async fn executables(
+    pub async fn executables_all_dependencies(
```

Comment:
sorry if it's nitpicking

maybe  we can name it 
```
pub async fn executables_from_all_dependencies
```

and 
```
pub async fn executables_from_direct_dependencies
```

?

Reply:
Done